_rules/audio-transcript-2eb176.md

```diff
@@ -5,6 +5,8 @@ rule_type: atomic
 description: |
   This rule checks that `audio` elements have a transcript that includes all auditory information.
 accessibility_requirements:
```

Comment:
Add SC 1.3.1 as a secondary requirement because it only applies when the transcript is visible.

Reply:
We did not include 1.3.1 as made the transcript as an assumption - I am not sure this should be done.

Reply:
Since we added Passed Ex 4, we would want to add 1.3.1 as secondary so an implementation that fails 1.3.1 is still consistent with this rule.